nautilus_trader/adapters/bybit/endpoints/trade/place_order.py

```diff
@@ -16,43 +16,21 @@
 import msgspec
 
 from nautilus_trader.adapters.bybit.common.enums import BybitEndpointType
-from nautilus_trader.adapters.bybit.common.enums import BybitOrderSide
-from nautilus_trader.adapters.bybit.common.enums import BybitOrderType
 from nautilus_trader.adapters.bybit.common.enums import BybitProductType
-from nautilus_trader.adapters.bybit.common.enums import BybitTimeInForce
-from nautilus_trader.adapters.bybit.common.enums import BybitTpSlMode
-from nautilus_trader.adapters.bybit.common.enums import BybitTriggerDirection
-from nautilus_trader.adapters.bybit.common.enums import BybitTriggerType
 from nautilus_trader.adapters.bybit.endpoints.endpoint import BybitHttpEndpoint
+from nautilus_trader.adapters.bybit.endpoints.trade.batch_place_order import BybitBatchPlaceOrder
 from nautilus_trader.adapters.bybit.http.client import BybitHttpClient
 from nautilus_trader.adapters.bybit.schemas.order import BybitPlaceOrderResponse
 from nautilus_trader.core.nautilus_pyo3 import HttpMethod
 
 
-class BybitPlaceOrderPostParams(msgspec.Struct, omit_defaults=True, frozen=True):
+class BybitPlaceOrderPostParams(
```

Comment:
Is this the right inheritance direction, or its only `BybitPlaceOrder` which needs the `category`?

Reply:
`BybitPlaceOrderPostParams` has `.category`.

`BybitBatchPlaceOrderPostParams` has `.category` and `.request`.